clean.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: clean [options]"
+    echo "Cleans the local dev environment."
+    echo
+    echo "  -b     Delete bin directory"
+    echo "  -p     Delete packages directory and NuGet package caches"
+    echo "  -a     Remove all untracked files in the working tree and delete NuGet package caches"
+    exit 1
+}
+
+# Parse arguments
+
+if [ $# == 0 ]
+then
+    echo "Please specify at least one option."
+    echo "Use 'clean -h' for help."
+    exit 1
+fi
+
+while [[ $# > 0 ]]
+do
+    opt="$1"
+    case $opt in
+        -h|--help)
+        usage
+        ;;
+        -b)
+        CleanBinDir=true
+        ;;
+        -p)
+        CleanPackagesDir=true
+        CleanPackagesCache=true
```

Comment:
You're right that it's not necessary with the current code.  I'm planning on changing it to have deleting the cache be a separate option, though, so it will be necessary now.